public/db_foods.JSON

```diff
@@ -21,5 +21,36 @@
         "cheese": "芝士",
         "cucumber": "黄瓜",
         "meat": "肉饼"
+    },
+    "sandwich": {
+        "bread": "面包",
+        "lettuce": "生菜",
+        "tomato": "西红柿",
+        "beef": "牛肉"
+    },
+    "pancake": {
+        "butter": "黄油",
+        "honey": "蜂蜜",
+        "pancake": "薄饼",
+        "plate": "盘子"
+    },
+    "pancake toppings": {
+        "fruits": "水果",
+        "yoghurt": "酸奶",
```

Comment:
Better use American English spelling: https://www.quora.com/Whats-the-correct-spelling-yogurt-or-yoghurt